index.html

```diff
@@ -3570,7 +3570,7 @@ <h3><dfn>Switch To Frame</dfn></h3>
       return <a>error</a> with <a>error code</a> <a>no such frame</a>.
 
      <li><p><a>Set the current browsing context</a>
-     with <var>element</var>’s <a>nested browsing context</a>.
+     to the [=node navigable=] of <var>element</var>.
```

Comment:
This isn't right. The node navigable of an element is the navigable it lives in. You want https://html.spec.whatwg.org/#nested-navigable , which should probably be exported.